### ISSUE TITLE
Overriding Mal. 

### ISSUE THREAD

**Issue by lokijul** [lap00zza/9anime-Companion#72]
**Select your issue type:** (check at least one)
- [x] Bug
- [ ] Question
- [ ] Suggestion
- [ ] Other (please describe):

**Describe your issue:**
Just now when I was going to watch One piece the 9anime player went directly to the first episode like usual(as it forgets that I watched OP before since I'm using a different device) but your system set it that I watched just ep 1 when in actuality I watched up to 806.

In other words, your thingy is overriding my "eps seen" on Mal then setting it like I haven't watched any episodes at all.

**Comment by lap00zza** [comment]
@lokijul thanks for reporting. Please keep an eye on this thread to check the progress.

**Comment by lap00zza** [comment]
Fixed in fbf289e. Shipped in version `1.2.0.`.
